tests/www/test_views.py

```diff
@@ -446,10 +446,6 @@ def test_xcom(self):
         resp = self.client.get(url, follow_redirects=True)
         self.check_content_in_response('XCom', resp)
 
-    def test_edit_dagrun_page(self):
```

Comment:
This was also on my todo list.
Actually, if you look closer :D you can see what my intention was. I wanted to edit [dagruns ](https://github.com/apache/airflow/blob/master/airflow/www/views.py#L2306) not the [dag](https://github.com/apache/airflow/blob/master/airflow/www/views.py#L2553).